apps/condo/domains/banking/access/BankIntegrationOrganizationContext.js

```diff
@@ -8,19 +8,22 @@ const get = require('lodash/get')
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 const { getById } = require('@open-condo/keystone/schema')
 
+const { BANK_INTEGRATION_IDS } = require('@condo/domains/banking/constants')
+const { checkBankIntegrationsAccessRights } = require('@condo/domains/banking/utils/accessSchema')
 const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
 const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
 const { SERVICE } = require('@condo/domains/user/constants/common')
 
 async function canReadBankIntegrationOrganizationContexts (args) {
-    const { authentication: { item: user }, originalInput, context } = args
+    const { authentication: { item: user } } = args
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isAdmin || user.isSupport) return true
 
     if (user.type === SERVICE) {
-        // TODO(VKislov): need to check access for service user with checkBankIntegrationsAccessRights(context, user.id, [bankIntegration])
-        return false
+        return {
+            integration: { accessRights_some: { user: user.id }, deletedAt: null },
```

Comment:
```suggestion
            integration: { accessRights_some: { user: user.id, deletedAt: null }, deletedAt: null },
```

Reply:
does not make sense, because if the user is deleted, the guard will work at the beginning and the request will be rejected without reaching this check